src/Router.tsx

```diff
@@ -20,12 +21,12 @@ export const ROUTES = {
   ADD_ARCHIVE: "/addarchive",
   POSTS: "/posts",
   POST_DETAIL: "/posts/:id",
+  ADD_POST: "/post",
```

Comment:
아래와 같은 느낌으로 표현되는게 더 명확할 것 같아요!
```suggestion
  ADD_POST: "/post/add",
```